API/TreeData/TreeDataMock.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+using MySql.Data;
+using MySql.Data.MySqlClient;
+
+namespace TreeData_CLI
+{
+    public class TreeRequest
+    {
+        public string Subquadrat { get; set; }
+        public string Tag { get; set; }
+        public string StemTag { get; set; }
+        public string SpCode { get; set; }
+        public double DBH { get; set; }
+        public string Codes { get; set; }
+        public string Comments { get; set; }
+        public bool WasDead(ArrayList Trees){
+            foreach(TreeStorage tree in Trees){
+                if(tree.Tag == Tag && tree.StemTag == StemTag
+                    && tree.Codes.Contains("dt") ){
+                    return true;
+                }
+            }
+            return false;
+        }
+    }
+    public class TreeStorage {

```

Comment:
TreeStorage should be in a file `WebApi/Model/Storage/TreeStorage.cs`

Reply:
Fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/0543202ea320047257a303f345157882127330c9

Reply:
Please move this file to the folder indicated above, and change the namespace accordingly - i.e. `ForestGEO.WebApi.Model.Storage`

Reply:
I think I got it fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/21e82c0ba5d7423824e06c9aab0cd7ae3aef2e80

Moving those files outside of the main directory for the project makes all of the references super long, but does organize the folders.